packages/headless/src/features/commerce/recommendations/recommendations-selector.ts

```diff
@@ -0,0 +1,32 @@
+import {isNullOrUndefined} from '@coveo/bueno';
```

Comment:
In general, should we test our selectors?

https://redux.js.org/usage/writing-tests#selectors

Reply:
Yeahhhhhh, I guess we should.